src/Model/Package.php

```diff
@@ -117,7 +117,7 @@ public function requireDefaultRecords()
         }
 
         /** @var QueuedJobService $jobService */
-        $jobService = QueuedJobService::singleton();
```

Comment:
Was this actually required? The service has the Injectable trait so should have this method available